nodejs/week3/homework-template/src/backend/api/meals.js

```diff
@@ -0,0 +1,85 @@
+const express = require("express");
+require("express-async-errors");
+const router = express.Router();
+const knex = require("../database");
+const { sum, limit } = require("../database");
+
+//to inser a new meal
+router.post("/", async (req, res) => {
+	const newMeal = {
+		title: req.query.title,
+		description: req.query.description,
+		location: req.query.location,
+		for_when: req.query.for_when,
+		max_reservations: req.query.max_reservations,
+		price: req.query.price,
+		created_date: req.query.created_date,
+	};
+	await knex("meals").insert(newMeal);
+	res.send(`new meal inserted: ${newMeal.title}`);
+});
+
+//to get a meal by id
+router.get("/:id", async (req, res) => {
+	const mealId = parseInt(req.params.id);
+	const meal = await knex("meals").where({ id: mealId });
```

Comment:
what happens if there's no meal with that id in the database?

how should a REST api respond when a resources is not found?